ts/Iterable.ts

```diff
@@ -353,7 +374,8 @@ class Iterable {
   * @param {string} link URL in string form to be either opened as an app link or as a normal one
   */
   static handleAppLink(link: string): Promise<boolean> {
-    console.log("handleAppLink")
+    Iterable.logger.log("handleAppLink")
+
     return RNIterableAPI.handleAppLink(link)
   }
 
```

Comment:
There is a console.log on line 397 that should be converted.

Reply:
Nice catch! It didn't select the correct one but I found it.